.travis.yml

```diff
@@ -44,9 +39,10 @@ matrix:
           env: STABILITY="dev"
 
     allow_failures:
-          # Latest dev is allowed to fail.
+        - php: 7.3
```

Comment:
This is not working for now, but the issue is on Travis' side. Will work as soon as they fix it.